app.js

```diff
@@ -0,0 +1,11 @@
+let calculators = ko.observableArray([new Calculator()]);
```

Comment:
This code has mainly been cut and pasted here from the previous javascript file for reasons described below